Sources/Charcoal/Filters/Stepper/StepperViewController.swift

```diff
@@ -51,8 +51,8 @@ public final class StepperFilterViewController: FilterViewController {
         ])
     }
 
-    public override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+    public override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
         topConstraint.constant = (view.frame.height - bottomButton.intrinsicContentSize.height) / 2
```

Comment:
Not sure why this is moved to `viewDidLayoutSubviews`, but I assume it fixes some issue 😄

Reply:
Yes, there was a spacing issue. 😄  I don't think it's safe to use the frames before `viewDidLayoutSubviews`, because they're not always correct.